Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,29 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const result =
+    // step 1.2.4
+    Math.floor(
+      100 *
+        tasks
+          // step 1.2.1
+          .map(x => x.duration / 60)
```

Comment:
Do not use name `x` unless you actually mean the algebraic 'unknown'. Using `x` here requires the reader of your code to mentally 'map' `x` to mean `task`. This is a burden for the reader. Your code becomes easier to understand if you are explicit in your naming, If you are mapping `tasks` then a single element is a `task`:

```js
.map(task => task.duration / 60)
```